### DISCUSSION TITLE
vocalisation type in BirdNET output

### DISCUSSION THREAD

**Discussion by Dubults** [birdnet-team/BirdNET-Analyzer#454] [Ideas]
Currently, the output from BirdNET is the name of the species detected and its confidence level (plus additional technical information depending on the output format selected). However, there are different types of bird calls - songs, alarm calls, flight calls, etc. I assume that all of these are used to train the BirdNET model. If so, BirdNET should be able to provide information on the type of sound in addition to the species name in its output of detections. This would be very important for researchers as it would provide more information on the status of the species at the given location. Also, the confidence of species identification could vary between different types of vocalisations and the researcher could choose different thresholds to use depending on the type of sound, rather than a single threshold for a species. I have checked the Discussions and Issues sections of the BirdNET-Analyzer github and have not seen anything about this. Am I the only one who thinks this would be a much needed feature to consider for future releases of BirdNET? Or have I missed something?

best wishes
Ainars

**Comment by whitneymonterey-png** [comment]
I strongly agree that this would be invaluable.... The data is already there I am guessing but needs to be classified by adding a new "Vocalization Type" data point of standardized nomenclature of vocalization types as perhaps used by the  Macaulay Library or by Xeno canto archives for example. I know for instance that Northern Cardinal vocalization types are sometimes sorted by 16 vocalization variations by ornithologist.   Is this somthing that could be crowd sourced? This might also be helpful in strengthening particular vocalizations/birds not as frequently  identified by BirdNet in North American North East such as Song Sparrows and various Wren species etc. I am an environmental educator and have advanced High School students that might be interested in contributing.  Love this tool... Whitney

**Reply by whitneymonterey-png** [reply]
Here is Xeno-Canto tag recommendations:

 grp:birds,
Type:
aberrant
call
     advertisement call, 
     agonistic call, 
     alarm call, 
     begging call,  
     defensive call,
     distress call
     flight call
     mating call, 
     nocturnal flight call, 
     release call, 
     social call
     territorial call
drumming, 
duet
echolocation
feeding buzz
imitation
mechanical sound, 
song
     calling song, 
     courtship song, 
     dawn song
     disturbance song
     female song,
     flight song, 
     rivalry song, 
     searching song,  
subsong, 

sex
     female, male

stage
      adult, juvenile, nestling, subadult

**Comment by whitneymonterey-png** [comment]
And here is Cornell/Macaulay Library via eBird search API terms for bird sounds:
Song

Call

Flight song

Flight call

Mechanical sound

Dawn song

Duet

People and Interviews
